src/map/script.c

```diff
@@ -9861,7 +9861,7 @@ BUILDIN_FUNC(monster)
 
 	if (script_hasdata(st, 9)) {
 		size = script_getnum(st, 9);
-		if (size > 3) {
+		if (size > SZ_BIG) {
 			ShowWarning("buildin_monster: Attempted to spawn non-existing size %d for monster class %d\n", size, class_);
```

Comment:
We should actually report the source npc here too, shouldn't we?
Actually on all of the checks in this function.